lib/src/se/llbit/chunky/PersistentSettings.java

```diff
@@ -110,9 +110,8 @@ private static void migrateOldSettings() {
     String lastTexturePack = settings.getString("lastTexturePack", null);
     if(lastTexturePack != null) {
       setLastTexturePack(lastTexturePack);
-      // TODO: remove setting with next release
-      // settings.removeSetting("lastTexturePack");
-      Log.info("migrated setting \"lastTexturePack\"");
+      // TODO: remove legacy setting with next release
```

Comment:
Can we tag this with `2.6.0` so its easy to search through all the files to find everything we need to remove?
```suggestion
      // TODO: Remove legacy setting in 2.6.0
```